src/objects/message.rs

```diff
@@ -2,6 +2,15 @@ use objects::User;
 use objects::Chat;
 use objects::MessageEntity;
 use objects::Contact;
+use objects::Document;
+use objects::Audio;
+use objects::Sticker;
+use objects::Video;
+use objects::PhotoSize;
+use objects::Voice;
+use objects::Location;
+use objects::Venue;
```

Comment:
Now that I think about it, could you change the notation to:

```rust
use objects:{...};
```

And then rust RustFmt on your project.